rabbitmq/rabbitmq-pod-monitor.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
do we want this to have its own ansible too or just instructions on kubectl apply-ing it?

Reply:
Not sure how we want to do this. @deepwinter set up the RabbitMQ cluster so he may have better insight.

Reply:
at a minimum just a line in a readme might be sufficient